__tests__/components/editor/PropertyComponent.test.js

```diff
@@ -62,31 +60,28 @@ describe('<PropertyComponent />', () => {
       it('renders the lookup component', () => {
         expect(wrapper.find('Connect(InputLookupQA)').length).toEqual(1)
       })
-
     })
   })
 
   describe('when there are no configuration values from the property template', () => {
-
     describe('for unconfigured templates of type:literal', () => {
       const template = {
-        "propertyURI": "http://id.loc.gov/ontologies/bibframe/heldBy",
+        "propertyURI": "http//:id.loc.gov/ontologies/bibframe/heldBy",
```

Comment:
what????  it's `http://` isn't it?

Reply:
@ndushay 	hahaha. small patch coming. :roll_eyes: